internal/pod_models.go

```diff
@@ -37,6 +37,17 @@ import (
 
 var processClassSanitizationPattern = regexp.MustCompile("[^a-z0-9-]")
 
+const (
+	// SidecarContainer is the name of the sidecar container
+	SidecarContainer string = "foundationdb-kubernetes-sidecar"
+	// MainContainer is the name of main container that runs FoundationDB
+	MainContainer string = "foundationdb"
+	// InitContainer is the name of the init container
+	InitContainer string = "foundationdb-kubernetes-init"
+	// SidecarDefaultImage is the name of the default sidecar and init container
+	SidecarDefaultImage string = "foundationdb/foundationdb-kubernetes-sidecar"
```

Comment:
We could also think about exposing this information to allow adopters to make use of these constants.